Test/comp/Extern3.js

```diff
@@ -15,5 +15,17 @@ let Library = (function() {
     }
   };
   
+  $module.Mixed = class Mixed {
```

Comment:
Does this change belong in this PR?

Reply:
Yes, the test file Extern.dfy is run with the C#, JavaScript, and Go compilers, so my change in Extern.dfy required me to also fill in handwritten code in all of Extern2.cs, Extern3.js, and Extern4.go.